index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Simple UI</title>
+        <link rel="stylesheet" href="./css/main.css" />
+        <link rel="stylesheet" href="./css/header.css" />
+        <link rel="shortcut icon" href="./images/simpleUIlogo.png" type="image/x-icon" />
+        <link rel="apple-touch-icon" href="./images/simpleUIlogo.png" />
+    </head>
+    <body>
+        <header class="header">
+            <a class="page-profile-link" href="/index.html">
+                <span class="page-profile">
+                    <img
+                        class="page-profile-logo"
+                        src="./images/simpleUIlogo.png"
+                        width="2em"
+                        alt=""
+                    />
+                    <span class="page-profile-title">Simple UI</span>
+                </span>
+            </a>
+            <a href="/pages/components/Avatar/avatar.html"
+                ><span class="docs-link">Documentation</span></a
+            >
+        </header>
+        <main class="main-container">
+            <img class="main-logo" src="./images/homeAnimation.gif" width="10em" alt="" />
+            <p class="main-desc">
+                An open source, simple, and customiseable library for building web applications.
+            </p>
+            <div class="button-home">
+                <a href="pages/howToUse.html"
+                    ><button class="main-start">Get Started</button></a
+                >
+                <a href="https://github.com/SalonSikdar/component-library"
+                    ><button class="main-github">GitHub</button></a
```

Comment:
No need to add a button tag inside the anchor tag. You can simply use the anchor tag and apply CSS to make it look like a button.
Ex: <a href="#">Get Started</a>
Google: Difference between anchor tag and button

Reply:
Thanks! the recommendation is noted, will design a component properly and implement the same in the next PR. keeping it as a button for now.